components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -4899,7 +4900,7 @@ public static String encryptPassword(String plainTextPassword) throws APIManagem
     public static boolean isValidWSDLURL(String wsdlURL, boolean required) {
         if (wsdlURL != null && !"".equals(wsdlURL)) {
             if (wsdlURL.startsWith("http:") || wsdlURL.startsWith("https:") ||
-                wsdlURL.startsWith("file:") || wsdlURL.startsWith("/registry")) {
+                    wsdlURL.startsWith("file:") || wsdlURL.startsWith("/registry")) {
```

Comment:
Please check the PR, there seems to be a multiple places where we can move strings to constants :) ?